Mathlib/Analysis/Calculus/Deriv/Add.lean

```diff
@@ -282,6 +282,16 @@ theorem differentiableOn_neg : DifferentiableOn 𝕜 (Neg.neg : 𝕜 → 𝕜) s
   DifferentiableOn.neg differentiableOn_id
 #align differentiable_on_neg differentiableOn_neg
 
+theorem not_differentiableAt_abs_zero : ¬ DifferentiableAt ℝ (abs : ℝ → ℝ) 0 := by
+  intro h
```

Comment:
This *might* be slightly clearer to an outside reader.
```suggestion
  by_contra h

Reply:
I think `intro` is preferred in Mathlib